x-pack/plugins/maps/public/routes/list_page/load_list_and_render.tsx

```diff
@@ -38,13 +39,9 @@ export class LoadListAndRender extends Component<Props> {
 
   async _loadMapsList() {
     try {
-      const results = await getSavedObjectsClient().find({
-        type: MAP_SAVED_OBJECT_TYPE,
-        perPage: 1,
-        fields: ['title'],
-      });
+      const results = await mapsClient.search();
```

Comment:
Can you call search with `{limit: 1 }`?

Reply:
Fixed in https://github.com/elastic/kibana/pull/153304/commits/9d8a30a94b813c30f599e62fccb01e423ce87fea